src/App.tsx

```diff
@@ -1,25 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
```

Comment:
The `getNumbers` function is imported but not used in this file. If it's not needed, consider removing the import to clean up the code.